src/Model/RefundType.php

```diff
@@ -30,4 +26,12 @@ public static function shipment(): self
     {
         return new self(self::SHIPMENT);
     }
+
+    /**
+     * @psalm-return mixed
+     */
+    public function getValue()
```

Comment:
Do we need it?

Reply:
Its using twice times in `RefundEnumType.php`

Reply:
Maybe we should cast the value to string?